src/components/splashScreen/TitleBadge.tsx

```diff
@@ -1,14 +1,36 @@
+import { useState } from "react";
 import styles from "./TitleBadge.module.css";
 
 type TitleBadgeProps = {
   isPaniniOrdered: boolean;
+  handleTransition: () => void;
 };
 
 export default function TitleBadge(props: TitleBadgeProps) {
+  const [isButtonAvailable, setIsButtonAvailable] = useState(true);
+  const handleClick = () => {
+    setIsButtonAvailable(false);
+    setTimeout(() => {
+      setIsButtonAvailable(true);
```

Comment:
**thought:** I don't think this logic is necessary, button should only run handleTransition handler in my opinion